zig build fails with no field named 'path' in union 'Build.LazyPath'

```
/home/runner/work/fiat-crypto/fiat-crypto/fiat-zig/build.zig:5:33: error: no field named 'path' in union 'Build.LazyPath'
/opt/hostedtoolcache/zig/master/x64/lib/std/Build.zig:2133:22: note: union declared here
```
https://github.com/mit-plv/fiat-crypto/actions/runs/9092827097/job/24990411932?pr=1911#step:4:5
cc @jedisct1

--- a/fiat-zig/build.zig
+++ b/fiat-zig/build.zig
@@ -2,7 +2,7 @@ const std = @import("std");
 
 pub fn build(b: *std.Build) void {
     const main_tests = b.addTest(.{
-        .root_source_file = .{ .path = "src/main.zig" },
+        .root_source_file = b.path("src/main.zig"),
     });
     const run_main_tests = b.addRunArtifact(main_tests);
     const test_step = b.step("test", "Run library tests");

--- a/inversion/zig/build.zig
+++ b/inversion/zig/build.zig
@@ -2,7 +2,7 @@ const std = @import("std");
 
 pub fn build(b: *std.Build) void {
     const main_tests = b.addTest(.{
-        .root_source_file = .{ .path = "inversion.zig" },
+        .root_source_file = b.path("inversion.zig"),
     });
     const run_main_tests = b.addRunArtifact(main_tests);
     const test_step = b.step("test", "Run tests");

